.github/workflows/pull-request.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         id: configure
         name: Set up environment
 
-      - uses: UWIT-IAM/actions/configure-gcloud-docker@0.1
+      - uses: UWIT-IAM/actions/configure-gcloud-docker-gcloud-v101@0.1.17
```

Comment:
So this image ships with Python 3.10, which is why you've updated pyproject.toml to allow for it?

Any concerns about issues with 3.9? or 3.11?